sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescription.java

```diff
@@ -96,14 +96,6 @@ public final class QueueDescription {
             namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
     private Integer maxDeliveryCount;
 
-    /*
-     * The maximum message size for a message.
-     */
-    @JacksonXmlProperty(
-        localName = "MaxMessageSizeInKilobytes",
-        namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
-    private Long maxMessageSizeInKilobytes;
-
     /*
      * Value that indicates whether server-side batched operations are enabled.
```

Comment:
Does the `TopicDescription` has the same problem?

Reply:
Nope, I check the `MaxMessageSizeInKilobytes` PR  https://github.com/Azure/azure-sdk-for-java/pull/25147 , in `TopicDescription` this field is added to the end.